CONTRIBUTING.md

```diff
@@ -46,7 +46,7 @@ facilitates social contribution, easy testing and peer review.
 
 To contribute a patch, the workflow is as follows:
 
-  1. Fork repository
+  1. Fork repository [only first time](https://help.github.com/en/articles/configuring-a-remote-for-a-fork).
```

Comment:
Can you change this to [kallewoof's suggestion](https://github.com/bitcoin/bitcoin/pull/17157#issuecomment-542567216) above.

```suggestion
  1. Fork repository ([only the first time](https://help.github.com/en/articles/configuring-a-remote-for-a-fork)).
```